RM_SUI/RM_SUI/ContentView.swift

```diff
@@ -8,12 +8,165 @@
 import SwiftUI
 
 struct ContentView: View {
+    @State var isOnToggle = false
+    @State var isAddingBalanceAlertShown = false
+    @State var balance = 0
+    @State var addedMainBalanceValue = ""
+    @State var addedAdditionalBalanceValue = ""
+    @State var isVisibleAccount = false
+    @State var isTransferAlertShown = false
+    
+    @State var newAccount = AccountManagement()
+    
     var body: some View {
         VStack {
-            Text("")
+            ZStack {
+                HStack {
+                    VStack(spacing: 15) {
```

Comment:
```suggestion
                    VStack(spacing: 15) {
```
я бы вынес в константы

Reply:
вынести